foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,44 +1,84 @@
 ### Introduction
 
-We're going to make an implementation of grade-school classic "rock paper scissors". If you don't know what that is check the [Wikipedia article](https://en.wikipedia.org/wiki/Rock%E2%80%93paper%E2%80%93scissors) or [this](https://www.wikihow.com/Play-Rock,-Paper,-Scissors) detailed step-by-step. For the moment we're just going to play the game from the browser console, but **we will revisit this project in a later lesson and add a Graphical User Interface with buttons and text,** so don't forget to keep the code on GitHub! You might notice some 'Live Preview' links in the student solutions that have a GUI - this is coming in a later lesson. When you get there don't forget to come back and add your link!
+For this project, you will create the game **Rock Paper Scissors**. This game will be played entirely on the console. To learn more, or if you're not familiar with the game, read this [Wikipedia article](https://en.wikipedia.org/wiki/Rock%E2%80%93paper%E2%80%93scissors) or this [detailed step-by-step guide.](https://www.wikihow.com/Play-Rock,-Paper,-Scissors)
 
-### Quick exercises before starting
+<div class="lesson-note">
+Some of the student solutions below contain buttons, text, and other elements. These elements are part of what is called a graphical user interface (GUI) You'll create the GUI in a later lesson. In the meantime, remember to commit your code to GitHub.
+</div>
+
+### Quick exercise before starting
+
+1. Identify three ways to include JavaScript in a webpage.
+1. Check if the webpage includes JavaScript:
+   - Write `console.log("Hello World")` in JavaScript.
+   - Check if "Hello World" is logged in the browser console once you open your webpage.
```

Comment:
This step feels a little out of place to me (the original too).
We have not yet instructed the learner to set anything up, only identify the 3 ways to include JS in a web page. Suddenly we're now saying "write JS and check if the web page runs it" - but we haven't made a web page yet.

I feel like the "identify three ways to include JS in a web page" bit might be better placed at the start of the assignment section itself, though I don't feel too strongly about this.

I do think the "test with a console log" but should at the very least come after we've instructed the learner to create a blank HTML with script tag. Though I wonder by this point, is it a necessary step? I don't feel strongly about its inclusion/exclusion but do thing it makes way more practical sense there and not before.

Reply:
Good point.
 
We can add the console log part as a third step in the initial structure step. I don't think "Identify three ways to include JavaScript in a webpage." step is necessary.